R/DataPulls.R

```diff
@@ -0,0 +1,137 @@
+# @file DataPulls.R
+#
+# Copyright 2020 Observational Health Data Sciences and Informatics
+#
+# This file is part of SkeletonComparativeEffectStudy
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+
+#' gets a table with data from preference score distribution.
```

Comment:
Nitpicking: start with a capital letter, and drop the 's' at the end of 'gets'. 

Also, 'table' and 'data' seem superfluous. When does a table contain something else than data? When does a function return something that is not data? I recommend:

'Get the preference score distribution'